src/20240916_Multi_RescueTokensFromAdapters/AaveV3Ethereum_RescueTokensFromAdapters_20240916.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {IERC20} from 'aave-v3-core/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+
+/**
+ * @title Minimal IRescuable
+ * @notice Defines the minimal basic interface for rescuable tokens mechanism
+ **/
+interface IRescuable {
+  /**
+   * @notice Emergency rescue for token stucked on this contract, as failsafe mechanism
+   * @dev Funds should never remain in this contract more time than during transactions
+   * @dev Only callable by the owner
+   * @param token The address of the stucked token to rescue
+   */
+  function rescueTokens(IERC20 token) external;
+}
+
+/**
+ * @title RescueTokensFromAdapters
```

Comment:
I think better to call the whole thing as ParaswapIncidentResolution or similar, but no big deal

Reply:
ok then I'll leave the naming as is so that I don't have to update all the text

Reply:
will leave as is, generator is throwing rate limit issues right now and don't want to migrate all the text manually